src/Components/Blazor/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -171,7 +172,10 @@
       Outputs="$(_BlazorLinkerOutputCache)">
 
     <ItemGroup>
-      <_BlazorDependencyAssembly Include="@(_BlazorDependencyInput)" IsLinkable="$([System.String]::Copy('%(FileName)').StartsWith('System.'))" />
+      <_BlazorDependencyAssembly Include="@(_BlazorDependencyInput)" />
+      <_BlazorDependencyAssembly IsLinkable="true" Condition="$([System.String]::Copy('%(Filename)').StartsWith('System.'))" />
+      <_BlazorDependencyAssembly IsLinkable="true" TypeGranularity="true" Condition="$([System.String]::Copy('%(Filename)').StartsWith('Microsoft.AspNetCore.'))" />
```

Comment:
Doesn't need to be as part of this PR, but we might want to think about having a way for the user to toggle the defaults.

Reply:
I agree. It would be nice if component libraries had an easy way to say "I'm safe to be linked". However it's out of scope for this PR.